internal/cli/root.go

```diff
@@ -169,6 +189,32 @@ func (c *Client) API(cmd *cobra.Command) *packngo.Client {
 	return c.apiClient
 }
 
+func (c *Client) MetalAPI(cmd *cobra.Command) *metal.APIClient {
+	if c.metalToken == "" {
+		log.Fatal("Equinix Metal authentication token not provided. Please set the 'METAL_AUTH_TOKEN' environment variable or create a configuration file using 'metal init'.")
+	}
+
+	if c.metalApiClient == nil {
+		httpClient := &http.Client{
+			Transport: &headerTransport{
+				header: getAdditionalHeaders(cmd),
+			},
+		}
+
+		err := c.metalApiConnect(httpClient)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+	return c.metalApiClient
+}
+
+func getMetalVersion() string {
```

Comment:
This can be removed; metal-cli doesn't need to know the version of the underlying SDK at runtime.